metasv/generate_final_vcf.py

```diff
@@ -95,7 +95,7 @@ def convert_metasv_bed_to_vcf(bedfile=None, vcf_out=None, vcf_template_file=vcf_
                 is_pass = (int(interval.fields[8]) != -1) and (svlen == 0 or svlen >= 100)
             sv_id = "."
             ref = "."
-            alt = [vcf.model._SV(sv_type)]
```

Comment:
Keep the first one "alt = [vcf.model._SV(sv_type)]" as that uses the pyvcf
module properly.

On Fri, Sep 4, 2015 at 1:44 PM, msahraeian <notifications@github.com> wrote:

> In metasv/generate_final_vcf.py
> <https://github.com/bioinform/metasv/pull/61#discussion_r38792236>:
>
> > @@ -95,7 +95,7 @@ def convert_metasv_bed_to_vcf(bedfile=None, vcf_out=None, vcf_template_file=vcf_
> >                  is_pass = (int(interval.fields[8]) != -1) and (svlen == 0 or svlen >= 100)
> >              sv_id = "."
> >              ref = "."
> > -            alt = [vcf.model._SV(sv_type)]
>
> I am confused. It seems that it is not only removed but also modified:
> It was originally
> "alt = [vcf.model._SV(sv_type)]"
> now it is
> "alt = ["<%s>" % sv_type]"
> which one should I keep?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bioinform/metasv/pull/61/files#r38792236>.
>